tests/io/test_cfradial2.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# Copyright (c) 2023-2025, openradar developers.
+# Distributed under the MIT License. See LICENSE for more info.
+
+import pytest
+import xarray as xr
+from open_radar_data import DATASETS
+
+import xradar as xd
+
+
+@pytest.fixture
```

Comment:
@syedhamidali Maybe my suggestion was a bit brief, but this fixture is already defined in conftest.py as well as the tmp_file fixture. You should be able to just use them here and can remove the re-definitions here.